lib/stat_tracker.rb

```diff
@@ -169,7 +175,15 @@ def fewest_tackles(season_id)
     highest_total = tackle_totals.min_by { |team, tackles| tackles}
     @teams.find { |team| team.team_id == highest_total.first}.team_name
   end
```

Comment:
not sure what this is doing here in lowest scoring visitor

Reply:
yes, sorry i dont know why that is there.